src/main/webapp/script.js

```diff
@@ -22,3 +21,129 @@ function getDatabases(){
   });
 }
 
+function submitDatabaseForm() { 
+  document.getElementById("database-select-form").submit();
+}
+
+//Get database and selected tables from query string and /tables-from-db
+function getDatabaseAndTable(){
+    const tablesUrl = '/tables-from-db';
+    const search = window.location.search;
+    const queryString = tablesUrl + search;
+    var startIndex = 0;
+    var databaseString= "";
+    
+    for (var i = 0; i<search.length; i++){
+        if (search.charAt(i) == '='){
+            startIndex = i;
+        }
+        else if (search.charAt(i) == '&'){
```

Comment:
Can move the else if up a line

if {
} else if {
} else {
}

Reply:
Done!